recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
+{% set sha256 = "a14c20c61c086414895142d22d4ba9b4a5267a743dddfff860d947a5126fdb79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-fixed.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-fixed.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.7*
+    - setuptools
+
+  run:
+    - python 2.7*
+    - frozendict
+    - enum34
+    - futures
+
+test:
+  imports:
+    - plum
+
+  commands:
+    - python -m unittest discover
+
+  source_files:
+    - test
+
+about:
+  home: https://bitbucket.org/aiida_team/plum/
+  license: GPLv3, MIT
+  license_file: LICENSE
+  summary: "A python workflow library that supports multithreading, and RabbitMQ as a message broker for starting/stopping/monitoring tasks."
+
+extra:
+  recipe-maintainers:
+    - ltalirz
+    - muhrin
```

Comment:
@muhrin -- Are you ok being added as a co-maintainer of the `plumpy` recipe on conda-forge? You were added to the recipe by @ltalirz.